lib/ui/common/components/cobble_sheet.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:cobble/ui/theme/with_cobble_theme.dart';
+import 'package:flutter/material.dart';
+
+/// Shows a modal Cobble design bottom sheet.
+///
+/// A modal bottom sheet prevents the user from interacting with the rest of
+/// the app until she dismisses it.
+///
+/// The [context] argument is used to look up the [Navigator] and [Theme] for
+/// the bottom sheet. It is only used when the method is called. Its
+/// corresponding widget can be safely removed from the tree before the bottom
+/// sheet is closed. Inside [builder] function you should use [context] provided
+/// to builder instead of one passed to [showCobbleSheet].
+Future<void> showCobbleSheet({
```

Comment:
This component isn't implemented correctly. In fact there are 2 bottom sheet:

1. opens over bottom navigation, must be dismissed by clicking on background or dragging on handle
2. opens above bottom navigation (inside "tabview"), is persistent, doesn't have background and can't be dismissed. Also doesn't have a handle.

Current implementation support 1. use case only.

Reply:
Implemented 2. use case, too.